webview/src/experiments/components/table/body/TableContent.test.tsx

```diff
@@ -0,0 +1,858 @@
+            Created: '2023-04-17T00:50:06',
+            branch: 'current',
+            commit: {
+              author: 'Matt Seddon',
+              date: '4 days ago',
+              message: 'Update dependency dvclive to v2.6.4 (#75)\n\n',
+              tags: []
+            },
+            deps: {
+              data: {
+                changes: false,
+                value: 'ab3353d'
+              },
+              'train.py': {
+                changes: false,
+                value: 'f431663'
+              }
+            },
+            description: 'Update dependency dvclive to v2.6.4 (#75)',
+            id: '48086f1',
+            label: '48086f1',
+            metrics: {
+              'training/metrics.json': {
+                step: 14,
+                test: {
+                  acc: 0.7735,
+                  loss: 0.9596208930015564
+                },
+                train: {
+                  acc: 0.7694,
+                  loss: 0.9731049537658691
+                }
+              }
+            },
+            params: {
+              'params.yaml': {
+                epochs: 15,
+                lr: 0.003,
+                weight_decay: 0
+              }
+            },
+            selected: false,
+            sha: '48086f1f70b2c535bafd830f7ce956355f6b78ec',
+            starred: false
+          },
+          subRows: []
+        },
+        {
+          depth: 0,
+          getAllCells: mockedGetAllCells,
+          getIsExpanded: mockedGetIsExpanded,
+          id: '3',
+          index: 3,
+          original: {
+            Created: '2023-04-17T00:49:44',
+            branch: 'current',
+            commit: {
+              author: 'Matt Seddon',
+              date: '4 days ago',
+              message: 'Drop checkpoint: true (#74)\n\n',
+              tags: []
+            },
+            deps: {
+              data: {
+                changes: false,
+                value: 'ab3353d'
+              },
+              'train.py': {
+                changes: false,
+                value: 'f431663'
+              }
+            },
+            description: 'Drop checkpoint: true (#74)',
+            id: '29ecaaf',
+            label: '29ecaaf',
+            metrics: {
+              'training/metrics.json': {
+                step: 14,
+                test: {
+                  acc: 0.7735,
+                  loss: 0.9596208930015564
+                },
+                train: {
+                  acc: 0.7694,
+                  loss: 0.9731049537658691
+                }
+              }
+            },
+            params: {
+              'params.yaml': {
+                epochs: 15,
+                lr: 0.003,
+                weight_decay: 0
+              }
+            },
+            selected: false,
+            sha: '29ecaaf3adf216045e96e81fb8e3027c9122af52',
+            starred: false
+          },
+          subRows: []
+        }
+      ]
+    })
+  } as unknown as Table<Experiment>
```

Comment:
[I] When we add the extension part of this it would be good to add more commits/at least one extra branch to the base test fixture.

Reply:
[Q] Would also mean we would get the design added to a Story, right?

Reply:
Yes, as soon as there is more than one branch, the new design appears. I think it might be good to add a story for it right away, so that it's possible to see it without faking it.